src/utils/dom.ts

```diff
@@ -0,0 +1,12 @@
+// Credit: https://github.com/tailwindlabs/headlessui/
+import type { ComponentPublicInstance } from '@vue/runtime-dom'
```

Comment:
TIL about this type! I've been searching for it for a while :grin:

Reply:
I've been searching for it, too! headlessui/vue used it.